### PR TITLE
Filling in TODOs for targeting

### PR DESCRIPTION
**Pull request by Ellmen**
This time without all the extra stuff.

### CODE REVIEW

File: models/targeting/nucleotide_insertion.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+# random insertion function
+# takes insertion size and produces random insertion string
+
+DNA_ALPHABET = "acgt"
```
Comment (jrhawley) [inline models/targeting/nucleotide_insertion.py]: this is part of the constants.py file. if you import it, you don't need this

File: models/targeting/probabilistic.py [hunk 2]
```diff
@@ -7,7 +7,6 @@
 # average time for SpyCas9 cutting in perfect match case
 average_cut_time = 60.0
 
-
```
Comment (mattsmart) [inline models/targeting/probabilistic.py:10]: put this newline back!

**Comment by mattsmart** [comment]
whats going on with indel_size.py?

**Comment by mattsmart** [comment]
I also think we shouldn't have a bunch of separate scripts for stuff like indel size, nt rand, and probability value -- all of them could be functions in probabilistic.py or a more aptly named script

File: models/targeting/nucleotide_insertion.py (hunk 1 above)
Comment (mattsmart) [inline models/targeting/nucleotide_insertion.py]: math3D constants.py is far away, I don't think we should import from such a distinct module

I'm ok having lone global constants for now, as we get more let's make a new constants.py or defaults.py in this module

**Comment by mattsmart** [comment]
we need some skeleton code to start filling in the genome_simulation.py script -- do you have plans for that in this PR @Ellmen ?

File: models/targeting/probabilistic.py (hunk 2 above)
Comment (Ellmen) [inline models/targeting/probabilistic.py:10]: oops I made some changes that I deleted and forgot about that

**Comment by Ellmen** [comment]
indel_size.py is the one I wasn't sure about. I seem to remember some talk about taking the distribution from the crispresso sample data? I agree about not having separate scripts it just said new modules in the TODOs but I can change that. I haven't done anything on genome_simulation yet but I can start working on that.

File: models/targeting/nucleotide_insertion.py (hunk 1 above)
Comment (jrhawley) [inline models/targeting/nucleotide_insertion.py]: right, this is a totally different set of scripts. my mistake

**Comment by mattsmart** [comment]
LGTM merging

**Pull request merged**